src/pep8ext_naming.py

```diff
@@ -412,3 +433,11 @@ def _extract_names(assignment_target):
         else:
             yield assignment_target.name
         return
+
+
+def is_mixed_case(name):
+    stripped_name = name.strip('_')
+    head = stripped_name[:1]
+    tail = stripped_name[1:]
+    if head.islower() and tail and not tail.islower() and not tail.isdigit():
+        return True
```

Comment:
It's a little odd that this function returns `True` or `None`. It's easy to always return a boolean:

```python
return head.islower() and tail and not tail.islower() and not tail.isdigit()
```

Reply:
Good point.